gateway/1.4/modules/ROOT/pages/flex-architecture-uc-overview.adoc

```diff
@@ -0,0 +1,103 @@
+= Integration Use Cases
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+The following integration use cases are determined by the ownership of the API Provider and API Consumer. Each use case shows physical patterns that demonstrate the necessary services in the system. 
+
+For the following use case diagrams, Flex Gateway is running in Connected Mode. However, you can also run Flex Gateway in Local Mode or use a combination of Connected Mode and Local Mode for separate runtimes in the same or in different environments.
+
+The use case diagrams use Kubernetes terminology, but you can extend the architecture to other technology stacks that have the appropriate network controls applied.
+
+Flex Gateway supports four use cases:
+
+* xref:flex-architecture-uc1.adoc[]
+* xref:flex-architecture-uc2.adoc[] 
+* xref:flex-architecture-uc3.adoc[] 
+* xref:flex-architecture-uc4.adoc[] 
+
+image:ad-intergration-use-cases.png[The four use cases are linked to products that they can support.]
+
+Each use case supports different integration patterns that vary depending on the location of APIs relative to the network in which Flex Gateway is located. You can use different use case patterns, depending on the different API owners and networks in your system. 
+
+In each use case, there are three different classifications of networks. The classifications describe the networks relation to your organization and the network Flex Gateway is in:
```

Comment:
"the networks relation to your organization" - do you mean "...related to your organization"?